src/resources/postcss/views/month/_calendar-event.pcss

```diff
@@ -7,6 +7,10 @@
 	.tribe-events-calendar-month__calendar-event {
 		margin: var(--spacer-1) 0;
 		position: relative;
+
+		&:first-of-type {
```

Comment:
https://www.loom.com/share/89aba7b05b52476c9b61804cc0a4d5cc

Reply:
So you're right in that case, but that's only one case. Other cases include: if we have a featured event (it has `--spacer-1` spacing above and below it), this could be below a multiday event too, if we have a regular event below a multiday event (it has a `--spacer-0`). What I mean by the wrapper around these single day events is that the `:first-of-type` selector will be relative to the `div` as a child of the `.tribe-events-calendar-month__day-events` wrapper, where the first single day event will always be the first of type.

So given all these cases, we need to think about whether we want the `.tribe-events-calendar-month__day-events` wrapper (and same for multiday) or not, and what the specific spacing cases will be.

Reply:
^ Agreed